### PR TITLE
feat: Provide initial KEDA scaler for HTTP add-on

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

File: artifacthub/add-ons-http/0.0.1/artifacthub-pkg.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+links:
+  - name: GitHub
+    url: https://github.com/kedacore/http-add-on
+readme: |
+  Alpha 1 is the first official release of the KEDA HTTP Add On, and is intended as a preview for developers. With this release, we're inviting community members to try out the technology, report bugs, and generally give us feedback.
+changes:
+  - Initial version
+maintainers:
+  - name: Sample maintainer
+    email: sample-maintainer@email.com
```
Comment (zroubalik) [inline artifacthub/add-ons-http/0.0.1/artifacthub-pkg.yml]: This should be set probably, right?

**Review by zroubalik — COMMENTED** [review]
Looking good, maintainer is missing though

File: artifacthub/add-ons-http/0.0.1/artifacthub-pkg.yml (hunk 1 above)
Comment (tomkerkhove) [reply artifacthub/add-ons-http/0.0.1/artifacthub-pkg.yml]: Removed it since it's not mandatory

Comment (tomkerkhove) [reply artifacthub/add-ons-http/0.0.1/artifacthub-pkg.yml]: https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml

**Review by zroubalik — APPROVED** [review]

**Pull request merged**